rocrand/PKGBUILD

```
@@ -34,4 +34,7 @@ package() {
 /opt/rocm/rocrand/lib
 EOF
   install -Dm644 "$srcdir/rocRAND-rocm-$pkgver/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+  # Some packages search for hiprand includes in /opt/rocm/include/hiprand
+  install -d "$pkgdir/opt/rocm/include/"
+  ln -s "/opt/rocm/hiprand/include/" "$pkgdir/opt/rocm/include/hiprand"
```

@fernandoblalves it is still not working for me. After building the package I cannot ls into the symlinked hiprand.
The following works.
```suggestion
  ln -s "/opt/rocm/rocrand/include/hiprand" "$pkgdir/opt/rocm/include/hiprand"
```

---

Shall I accept your commit suggestion?

---

Yeah go ahead, I'll merge the PR right after.